### PR TITLE
Consolidate localization

### PR DESCRIPTION
**Pull request by nagmat84**
While I have been working on some other parts, I realized that our localization was out-of-sync (mostly between the frontend and the backend). For example the frontend used `USER_DELETED` but the backend provided a constant `USER_REMOVED`. Hence, the localization did not work. Then I realized, that the order of variables was not consistent between the different languages, etc., pp. I freaked out and so I decided to clean it up from the scratch.

 * _All_ localization files of the backend `app/Locale` a are now 513 lines long and have the identical constants in identical positions :nerd_face: 
   _Note:_ I did not aim for a logic or semantic ordering, I mostly took the English localization as the ground truth and re-sorted the other files accordingly. There does not seem to be a reasonable ordering, so be it.
 * I checked which constants are actually used by the frontend but have missing either in the backend or the frontend, I added these constants
 * I checked which constants are were not used anymore, I removed these variables.

While I was on it, I improved typography as well :nerd_face: :

 * Proper quotation marks for the various languages, cp. https://en.wikipedia.org/wiki/Quotation_mark#Summary_table
 * Correct question and exclamation marks in Chinese; Chinese uses fixed-width marks which have different unicode char point than western symbols
 * Proper apostrophe to indicate left-out characters
 * Correct French spacing with non-breaking narrow space before '?' and '!' as well as non-breaking normal-width space before ':'
 * Correct horizontal ellipsis instead of three dots

### CODE REVIEW

**Comment by nagmat84** [comment]
I nearly forgot:

Actually I wanted to rephrase the options in the Album Protection Policy Dialog to make unambiguously clear to what these options apply. Previously we had a mixed and imprecise wording using the terms "user" or "people". So the dialog now look like this

Screenshot 2022-08-28 at 21-34-20

And then the disaster took its course ...

**Comment by ildyria** [comment]
LGTM once the proposition are merged. :)

**Comment by nagmat84** [comment]
I merged @ildyria's PRs into this one.

**Review by ildyria — APPROVED** [review]
LGTM

File: app/Locale/Polish.php
Comment (kamil4) [inline app/Locale/Polish.php]:
```suggestion
			'DELETE_ALBUM_CONFIRMATION' => 'Czy na pewno chcesz usunąć album „%s” razem z zawartością ? Ta akcja jest nieodwracalna!',
```

File: app/Locale/Polish.php
Comment (kamil4) [inline app/Locale/Polish.php]:
```suggestion
			'DELETE_TAG_ALBUM_CONFIRMATION' => 'Are you sure you want to delete the album „%s” (any photos inside will not be deleted)? This action can’t be undone!',
```

File: app/Locale/Polish.php
Comment (kamil4) [inline app/Locale/Polish.php]:
```suggestion
			'DELETE_UNSORTED_CONFIRM' => 'Czy na pewno usunąć wszystkie zdjęcia z „Nieposortowane”? Ta operacja nie może zostać cofnięta!',
```

File: app/Locale/Polish.php
Comment (kamil4) [inline app/Locale/Polish.php]:
```suggestion
			'ALBUM_MERGE' => 'Czy na pewno połączyć ten album „%1$s” z albumem „%2$s”?',
```

File: app/Locale/Polish.php
Comment (kamil4) [inline app/Locale/Polish.php]:
```suggestion
			'ALBUMS_MERGE' => 'Czy na pewno połączyć zaznaczone albumy z „%s”?',
```

File: app/Locale/Polish.php
Comment (kamil4) [inline app/Locale/Polish.php]:
```suggestion
			'ALBUM_MOVE' => 'Czy na pewno przenieść album „%1$s” do albumu „%2$s”?',
			'ALBUMS_MOVE' => 'Czy na pewno przenieść zaznaczone albumy do albumu „%s”?',
```

File: app/Locale/Polish.php
Comment (kamil4) [inline app/Locale/Polish.php]:
```suggestion
			'PHOTO_DELETE_CONFIRMATION' => 'Czy na pewno usunąć zdjęcie „%s”? Akcja jest nieodwracalna!',
```

**Review by kamil4 — CHANGES_REQUESTED** [review]
I admire your dedication. This definitely needed to be cleaned up. However, moving forward, I wonder if we could automate it somehow? It's a royal PITA to have to manually update every single language file when adding some new strings. No wonder they get out of sync.

**Comment by nagmat84** [comment]
**DON'T MERGE YET!**

I created this pull request in the course of my attempt to migrate the frontend to the new implementation of the modal dialog and my currently on-going task to refactor and improve the sharing of albums as in https://github.com/LycheeOrg/Lychee/issues/1462.

While I have been doing so, I needed some translations, I noticed that the files were mostly out-of-sync and then I thought it would be nice to pull this one out as an independent PR.

But as it turned out (I have just noticed), the old modern dialog is not well suited to deal with non-ASCII, multi-byte UFT-8 characters. (Surprise!) In other words, this PR can only be merged in combination with my upcoming PR.

**Comment by nagmat84** [comment]
> I admire your dedication. This definitely needed to be cleaned up. However, moving forward, I wonder if we could automate it somehow? It's a royal PITA to have to manually update every single language file when adding some new strings. No wonder they get out of sync.

Unfortunately, not now. The main obstacle is that we use the same localization for two different programming languages, i.e. PHP in the backend as part of [resources/views](https://github.com/LycheeOrg/Lychee/tree/master/resources/views), and JS for the frontend. There are localization tools which help you to automate things, but only for a _single_ language (at least as far as I know). I am already glad that @qwerty287 found this `sprintf` package for JS which allows us to use string replacements in PHP and JS in identical ways.

**Comment by nagmat84** [comment]
I resolved all merge conflicts with current master. Now I have to step over all files again and check whether all languages are still in sync after the conflict resolution. (Especially, the Spanish locale was challenging.)

@kamil4 You had some change requests regarding the Polish translation. I already committed your change requests a long time ago. Please re-check if you are fine with them.

**Review by kamil4 — APPROVED** [review]
LGTM (not tested).

**Pull request merged**
